fre/make/tests/null_example/test_checkout_null_yaml.py

```diff
@@ -0,0 +1,45 @@
+#tests for the create-checkout step of fre-make, for null_model.yaml
+import os
+from fre import fre
+from pathlib import Path
+from fre.pp import configure_script_yaml as csy
+from click.testing import CliRunner
+import subprocess
+from fre.make import createCheckout
+
+# Set example yaml paths, input directory
+test_dir = Path("fre/make/tests/null_example")
+yamlfile = Path(f"{test_dir}/null_model.yaml")
+
+#set platform and target
+platform = "ncrc5.intel"
+target = "debug"
+
+#set output directory
+#out_dir = Path(f"fre/make/tests/null_example/fre_make_out")
+#Path(out_dir).mkdir(parents=True,exist_ok=True)
```

Comment:
In my opinion, I feel like it would be nice it each test had output that was easy to find. Here, for the checkout, maybe we can set OUT to be:
```
OUT = f"{TEST_DIR}/checkout_out"
```
Where test_dir would be what I commented above. Also add in the out location to the `.gitignore` file